x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_modal/index.tsx

```diff
@@ -97,12 +98,16 @@ const ModalBodySection = styled.section`
   `}
 `;
 
+export type Alert = {
```

Comment:
This type is exported in 2 different places in this PR. We should move it somewhere common, if possible.